docs/scenarios/stacks.feature

```diff
@@ -1,99 +1,62 @@
-    And I can see an error message “this item is not currently available to request”
-
-
-Feature: As a library member of staff I need to know how many items I can request
-
-  Scenario: A logged in user checks their account to see how many items they can request
-
-    Given that I have logged in
-    When I click on my account
-    Then I can see a number indicating how many items I can request (20)
-
-
-Feature: I need to know which part of an archive record I can request
-
-  Scenario: A user has searched for a record from the Archives and Manuscripts collection and wants to see what they can request
-
-    Given I have selected an archive/manuscript
-    When I click to request it
-    Then I need to see what parts of the archive record are available to request
-    And how many parts I can request at any one time
-
-Feature: As a library member of staff that is logged in, I want to place a request for one item
-
-  Scenario: A logged in user searches for a work and places a request for one work/item
-
-    Given that I am on a work page
-    When I click the ‘Request It’ button
-    Then I see a pop up modal that shows the item I am requesting
-    And I can see how many requests I have left
-    Then I can click a button to confirm my request
-    And see a ‘request confirmed’ message
-
-
-Feature: As a library member of staff that is logged in, I want to place a request for multiple items from one work
-
-  Scenario: A logged in user places a request for multiple items within a work
-
-    Given that I am on a work page that has multiple items
-    When I have selected a number of those items to request
-    And have clicked a ‘Request It’ button
-    Then I see a pop up modal that shows an editable list of the items I’m requesting
-    And how many requests I have left
-    Then I can click a button to confirm my request
-    And see a ‘request confirmed’ message
-
-
-Feature: As a library member of staff that is logged in, I want to place a request at more than one time
-
-  Scenario: A logged in user places a request for different items at several times during a day/week
-
-    Given that I have requested an item previously
-    When I want to select more items to request
-    Then I want to see what items I’ve requested so far
-    And how many more I can request before I hit the limit
-
-
-Feature: As a library member of staff that is logged in, I want to place a request for multiple items at more than one time
-
-  Scenario: A logged in user places a request for multiple items at several times during a day/week
-
-    Given that I have already requested items previously
-    When I want to select more items to request at a later time
-    Then I need to see if an item has already been requested
-    And I need to see what I’ve requested so far
-    And how many more I can request before I hit the limit
+Feature: 1. As a library member I want to know if an item is available for requesting
+
+  Scenario: A user viewing a work page
```

Comment:
Wondering if we keep everything in the first person (as long as it makes sense to do so).

i.e. `Scenario: I'm viewing a work page`

Reply:
Agreed about using first person